flexget/plugins/filter/crossmatch.py

```diff
@@ -94,10 +97,31 @@ def entry_intersects(self, e1, e2, fields=None):
             log.trace('v1: %r' % v1)
             log.trace('v2: %r' % v2)
 
-            if v1 == v2:
-                common_fields.append(field)
+            if mode == "flexible":
+                v1Re = v1.lower()
+                v1Re = unicodedata.normalize('NFKD', v1Re).encode('ASCII', 'ignore')
+                v1Re = re.sub("[^a-zA-Z0-9 ]", " ", v1Re)
+                v1Re = v1Re.replace(' ','.*')
```

Comment:
I can add a extra mode

mode: standard (default)
mode: flexible (new mode)
mode: no_greedy

In no_greedy I'll replace spaces with .?, this will match Spider Man, Spider-Man and SpiderMan, but not "Spiders Are Man Best Friend".